File: 3379-score-of-a-string/score-of-a-string.py
class Solution:
    def scoreOfString(self, s: str) -> int:
        sum=0
    
        for i in range (1,len(s)):
            sum2= ord(s[i-1])-ord(s[i])
            if(0>sum2):
                sum2=sum2*-1
            sum+=sum2
               
        return sum

File: 3379-score-of-a-string/score-of-a-string.cpp
class Solution {
public:
    int scoreOfString(string s) {
        int sum=0,ans=0;
        for (int i=1;i<s.size();i++){
            sum=s[i-1]-s[i];
            if (0>sum){
                sum= sum*-1;
            }
            ans+=sum;
        }
        return ans;
    }
};